schedule/week_2/xquery_1.md

```diff
@@ -13,9 +13,9 @@ We will explore XQuery inside eXist-db, an open-source XML database and XQuery e
 
 We will use eXide, the integrated development environment (IDE) bundled with eXist-db. 
-After you installed and launched eXist-db, you can launch eXide by clicking on the eXide icon in your eXist dashboard. If you are running eXist on your own machine with default settings, your dashboard is at <http://localhost:8080> and you can launch eXide directly, bypassing the dashboard, at <http://localhost:8080/exist/apps/eXide/>.
+After you install and launch eXist-db, access eXide by clicking on the eXide icon in your eXist dashboard. If you are running eXist on your own machine with default settings, your dashboard is at <http://localhost:8080> and you can launch eXide directly, bypassing the dashboard, at <http://localhost:8080/exist/apps/eXide/>.
 
-When you install eXist-db on your own machine, you also install all documentation for both the XQuery function library (core XPath and XQuery functions) and eXist-db itself. The default location for the function library documentation on your local machine is <http://localhost:8080/exist/apps/fundocs/> (if you’re consulting local documentation for the first time, you’ll be prompted to generate it first), and you can also find a copy on the eXist-db demo server at <http://demo.exist-db.org/exist/apps/fundocs/> inside the database. You also run eXide and use interactive tutorials on the demo server any time you like; eXide is located at <http://demo.exist-db.org/exist/apps/eXide/index.html>. You can’t, for obvious reasons, save your own documents in the demo server, so if you want to do that, install your own copy of eXist-db on your own machine.
+When you install eXist-db on your own machine, you also install all documentation for both the XQuery function library (core XPath and XQuery functions) and eXist-db itself. The default location for the function library documentation on your local machine is <http://localhost:8080/exist/apps/fundocs/> (if you’re consulting local documentation for the first time, you’ll be prompted to generate it first). You can also find a copy on the eXist-db demo server on the internet at <http://demo.exist-db.org/exist/apps/fundocs/>. Indeed, you can access eXide and use interactive tutorials online on the demo server any time you like; that instance of eXide is located at <http://demo.exist-db.org/exist/apps/eXide/index.html>. eXist-db is often installed on web servers to support XML-based projects online, and if you have access to a web server, you may wish to explore the [advanced eXist-db documentation](https://exist-db.org/exist/apps/doc/advanced-installation.xml#headless) to try installing eXist-db there and learn how to use it to support dynamic searches. For now, though, unless you have permissions to save files on the web installations of eXist-db, you can’t upload and save your own documents there. That is why we recommend installing your own local copy of eXist-db on your machine, so you can practice and experiment with saving and querying your own files.
```

Comment:
This is where I added some introductory perspective on web server installations of eXist-dB.